app/views/artefacts/withdraw.html.erb

```diff
@@ -0,0 +1,29 @@
+<%= content_for :page_title, "Withdraw - #{@artefact.name}" %>
+<%= render partial: "artefact_header" %>
+
+<div class="callout callout-warning add-bottom-margin">
+  <div class="callout-title">
+    Warning
+  </div>
+  <div class="callout-body">
+    Withdraw a page from GOV.UK. This can’t be undone.
+  </div>
+</div>
+
+<div class="well">
+  <%= semantic_bootstrap_nested_form_for(@artefact, :html => { :class => 'artefact', :id => 'withdraw_artefact'}) do |f| %>
+    <h3 class="remove-top-margin add-bottom-margin">Withdraw</h3>
+      <div class="form-group">
+        <%= f.input :redirect_url, class: "form-control input-md-6",
+          label: ('Redirect to <span class="normal">(optional)<br />' +
+                  'For example: <code>/government/organisations/hm-revenue-customs</code></span>').html_safe%>
+        <%= button_to 'Withdraw',
```

Comment:
I'm not sure you want `button_to` here.  According to the docs, that creates a form with a single button within it, resulting in one form within another.

I also don't think there's an integration test that covers withdrawing and redirecting an artefact.  Might be worth adding one under `test/integration` (the cucumber is being gradually deprecated).